frontend/server/libs/dao/Assignments.dao.php

```diff
@@ -103,6 +103,29 @@ public static function getAssignmentForProblemset($problemset_id) {
         return new Assignments($row);
     }
 
+    final public static function getByIdWithScoreboardUrls($assignmentId) {
+        $sql = '
+                SELECT
+                   ps.scoreboard_url,
+                   ps.scoreboard_url_admin
+                FROM
+                    Assignments a
+                INNER JOIN
+                    Problemsets ps
+                ON
+                    ps.problemset_id = a.problemset_id
+                WHERE
+                    a.assignment_id = ? LIMIT 1;';
+        $params = [$assignmentId];
+
+        global $conn;
+        $rs = $conn->GetRow($sql, $params);
```

Comment:
nit:

```suggestion
        $ow = $conn->GetRow($sql, [$assignmentId]);
```

por consistencia con la función de arriba.

Reply:
listo